js/apps/system/_admin/aardvark/APP/frontend/js/views/settingsView.js

```diff
@@ -260,13 +260,20 @@
             } else {
               var wfs = data.waitForSync;
               if (data.replicationFactor && frontendConfig.isCluster) {
-                if (data.replicationFactor === 'satellite' || data.distributeShardsLike !== undefined) {
+                var msg = null;
+                if (data.replicationFactor === 'satellite') {
+                  msg = 'This collection is a SatelliteCollection.';
+                } else if (data.distributeShardsLike !== undefined) {
+                  msg = 'This collection uses \'distributeShardsLike\'.';
```

Comment:
Maybe we should also mention which collection is the leading collection and that changes to the replication factor etc can be done there.

Reply:
Addressed!